openreview/stages/venue_stages.py

```diff
@@ -539,7 +539,9 @@ def __init__(self,
         remove_fields = [],
         rating_field_name = 'rating',
         confidence_field_name = 'confidence',
-        process_path = None
+        process_path = None,
+        content_query = {},
+        child_invitations_name = 'Official_Review'
```

Comment:
I think `name` should represent the child invitation name.

Reply:
we should have a parent_name that is optional and by default it is the name value